packages/SketchMorph2-Core.package/M2Backend.class/instance/userActionClickAt..st

```diff
@@ -1,5 +1,5 @@
 undo
 userActionClickAt: aPoint
-	(self isInTextMode or: [self isInEditingTextMode]) 
-		ifFalse: [self createUserActionWithStart: aPoint].
+	self context isInAnyDrawingMode 
```

Comment:
This is unnecessary since the viewport always checks this condition before calling the method.

Reply:
Updated in [9bc3233](https://github.com/hpi-swa-teaching/SketchMorph2/pull/85/commits/9bc3233bff6f481e33c33ecc46f8efe0587d16b4).

Reply:
This should also apply to similar methods like `M2Backend>>userActionStopDrawing`.